src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -362,6 +488,7 @@ private abstract class SendOperation : WriteOperation
         private sealed class BufferMemorySendOperation : SendOperation
         {
             public Memory<byte> Buffer;
+            internal MemoryHandle PinHandle;
```

Comment:
These handles are only used while performing the operation against AIO. They could be in a `List<MemoryHandle>` that is populated during the use of AIO, instead of per operation.

With a single handle, multi buffer sends/receives can't be performed using AIO.